### DISCUSSION TITLE
I can not display conda environment.

### DISCUSSION THREAD

**Discussion by ashixu** [starship/starship#5366] [Q&A] [answered]
my device is archlinux. and have installed anaconda.
my config file is following.
[starship (copy 1).txt](https://github.com/starship/starship/files/12271681/starship.copy.1.txt)

**Comment by mickimnet** [comment, marked as answer]
You need to put the `[conda]` module in the `format` section. Besides the status of `disabled`, the module needs to be included in the `format` section. For further information, see: [Default Prompt Format](https://starship.rs/config/#default-prompt-format).

**Reply by ashixu** [reply]
Oh, thank you. that's ok！I didn't see it 😂😂
